// @mastra/core/dist/provider-registry-4PH2JPIA.cjs
'use strict';

var chunk3YMDR4OL_cjs = require('./chunk-3YMDR4OL.cjs');



Object.defineProperty(exports, "GatewayRegistry", {
  enumerable: true,
  get: function () { return chunk3YMDR4OL_cjs.GatewayRegistry; }
});
Object.defineProperty(exports, "PROVIDER_MODELS", {
  enumerable: true,
  get: function () { return chunk3YMDR4OL_cjs.PROVIDER_MODELS; }
});
Object.defineProperty(exports, "PROVIDER_REGISTRY", {
  enumerable: true,
  get: function () { return chunk3YMDR4OL_cjs.PROVIDER_REGISTRY; }
});
Object.defineProperty(exports, "getProviderConfig", {
  enumerable: true,
  get: function () { return chunk3YMDR4OL_cjs.getProviderConfig; }
});
Object.defineProperty(exports, "getRegisteredProviders", {
  enumerable: true,
  get: function () { return chunk3YMDR4OL_cjs.getRegisteredProviders; }
});
Object.defineProperty(exports, "isProviderRegistered", {
  enumerable: true,
  get: function () { return chunk3YMDR4OL_cjs.isProviderRegistered; }
});
Object.defineProperty(exports, "isValidModelId", {
  enumerable: true,
  get: function () { return chunk3YMDR4OL_cjs.isValidModelId; }
});
Object.defineProperty(exports, "parseModelString", {
  enumerable: true,
  get: function () { return chunk3YMDR4OL_cjs.parseModelString; }
});
